src/test/java/dk/kb/util/other/StringListUtilsTest.java

```diff
@@ -0,0 +1,155 @@
+package dk.kb.util.other;
+
+import org.hamcrest.CoreMatchers;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class StringListUtilsTest {
+    
+    @Test
+    void truncateMiddle() {
+        
+        String string
+                = "SU1:SU1_DESK:HOLD,MOVE:HFBOT,HFGEO,HFPAN,HFZM,HFFAR,HFJTS,HFJSP,HFKVT,HAUDL,HATSS,HFFJ,HFTSS,LFZM,LFPAN,LFBOT,LFGEO,LFJTS,LFKVT,LFZM,LFFJ,HASEM,BAGEO,BAIVT,BALSA,BAPEN,BAHBS,BAINF,BAINH,BASEM";
+        
+        String result = StringListUtils.truncateMiddle(string, 100);
+        assertThat(result.length(), is(100));
+        assertThat(result,
+                   is("SU1:SU1_DESK:HOLD,MOVE:HFBOT,HFGEO,HFPAN,HFZM,HF...M,BAGEO,BAIVT,BALSA,BAPEN,BAHBS,BAINF,BAINH,BASEM"));
+    }
+
+    @Test
+    void truncateMiddleNoOp() {
```

Comment:
I have added failing unit tests to demonstrate where the documentation differs from behaviour. Please adjust either the documentation or the behaviour.

Reply:
Fixed the behaviour. 
```java
        if (string.length() < maxLength) {
```
=>
```java
        if (string.length() <= maxLength) {
```